### PR TITLE
Trib 235

### PR DESCRIPTION
**Pull request by Jayj30pro**
Added delete feature to save method

### CODE REVIEW

File: src/test/java/com/savvato/tribeapp/services/UserRoleMapServiceImplTest.java [hunk 1]
```diff
@@ -24,9 +24,16 @@
...
 import static org.mockito.Mockito.*;
 
+
+
 @ExtendWith(SpringExtension.class)
 public class UserRoleMapServiceImplTest extends AbstractTestConstants {
 
+  public static final String ROLE_ADMIN = "ROLE_admin";
+  public static final String ROLE_ACCOUNTHOLER =  "ROLE_accountholder";
+  public static final String ROLE_PRASEREVIEWER = "ROLE_phrasereviewer";
```
Comment (haxwell) [inline src/test/java/com/savvato/tribeapp/services/UserRoleMapServiceImplTest.java:34]: Minor typo here.. pHrasereviewer

File: src/main/java/com/savvato/tribeapp/services/UserRoleMapService.java [hunk 2]
```diff
@@ -5,7 +5,8 @@
 
 public interface UserRoleMapService {
 	
-	enum ROLES { ADMIN, ACCOUNTHOLDER, PHRASE_REVIEWER }
+	// enum ROLES { ADMIN, ACCOUNTHOLDER, PHRASE_REVIEWER }
```
Comment (haxwell) [inline src/main/java/com/savvato/tribeapp/services/UserRoleMapService.java:8]: We can remove this now.. it's no longer current.

File: src/test/java/com/savvato/tribeapp/services/UserRoleServiceImplTest.java [hunk 3]
```diff
@@ -40,3 +40,4 @@ public void getRoles() {
     }
 
 }
+
```
Comment (haxwell) [inline src/test/java/com/savvato/tribeapp/services/UserRoleServiceImplTest.java]: Hmm... I was trying to remove this blank line.. so you may see some commits from me attempting that.. never mind, however.

**Comment by haxwell** [comment]
@Jayj30pro **Be sure your commit messages are meaningful!** The last four commits, three of which were related to a migration script, contain the same comment: They should either be in one commit with the one message, or each have descriptive messages of their own. The fourth commit, is not related to the other three, but it has the same commit message. **Be careful!** Also, the commit message itself is not really helpful. You wouldn't be able to look at that a year from now and understand anything about the commit.

File: src/test/java/com/savvato/tribeapp/services/UserRoleMapServiceImplTest.java (hunk 1 above)
Comment (Jayj30pro) [reply src/test/java/com/savvato/tribeapp/services/UserRoleMapServiceImplTest.java:34]: Done

**Comment by haxwell** [comment]
Hey @Jayj30pro ... all that's left here for now is to clear up the merge conflicts. For some reason, it won't let me do it in the PR. So ball's in your court..

**Pull request closed without merging**